taichi/transforms/compile_to_offloads.cpp

```diff
@@ -218,6 +218,13 @@ void offload_to_executable(IRNode *ir,
     irpass::analysis::verify(ir);
   }
 
+  if (arch_is_cpu(config.arch)) {
```

Comment:
This pass has exactly the same behavior with what we have done implicitly in the runtime. Just move it to a more explicit place so that LLVM could see it. Maybe I should add a compilation config switch?